index.html

```diff
@@ -58,5 +64,185 @@ <h3 class="dev-intro">I'am a Software Developer</h3>
         </li>
       </ul>
     </section>
+
+    <!-- WORK SECTION -->
+
+    <section class="main-wrap">
+      <article class="work-section">
+        <div class="work-title">
+          <h2>My Recent Works</h2>
+          <div class="rule"></div>
+          <div class="recent-rule"></div>
+        </div>
+        <div class="main-article">
+          <div class="article-img">
+            <img class="hide-desktop" src="image/Tonic.png" alt="Tonic Image" />
+            <!-- <img class="hide-mobile" src="image/tonic-desktop.png" alt="" /> -->
```

Comment:
- [ ] [REQUIRED]  Please remove any line of code that you are not using as a good practice of making your code clean.😊